std/conv.d

```diff
@@ -5024,6 +5024,47 @@ T emplace(T, Args...)(void[] chunk, auto ref Args args)
     return result;
 }
 
+/**
+Given a $(D T) of $(D class), constructs an object of $(D class)
+type $(D T) at that address. The constructor is passed the arguments
+$(D Args). 
+
+This function can be $(D @trusted) if the corresponding constructor of
+$(D T) is $(D @safe).
+
+Returns: If $(D obj) is null, it will return a null reference of type $(D T).
+Otherwise, returns a reference to a newly constructed $(D T) at $(D obj)'s location.
+ */
+T emplace(T, Args...)(T obj, auto ref Args args)
+    if (is(T == class))
+{
+    if(obj is null)
```

Comment:
space after `if` (violation not only of phobos style but also of style two lines before)